### DISCUSSION TITLE
Would multiple cpu speed up lammps simulation ?

### DISCUSSION THREAD

**Discussion by unknwon-man** [mir-group/pair_nequip#59] [Q&A]
I am trying to speed up LAMMPS MD simulations in gpu environment.
I know that gpu works for force-calculation during LAMMPS. Would it speed up MD if I run one gpu + multiple cpus? even though cpu won't work for force-calculation...

Thank you for your help ..! 

(I'm trying not to use Allegro available with multi-gpu. Training is too poor for my system.)
(I am afraid if this question is more related to the LAMMPS. If so, I will try to search in LAMMPS community. Let me know.)

**Comment by Linux-cpp-lisp** [comment]
Hi @unknwon-man ,

Thanks for your interest in our code. If the model is running on the GPU, adding additional CPU resources is not expected to make any notable difference on performance.
